cfme/fixtures/utility_vm.py

```diff
@@ -32,29 +32,40 @@
 from cfme.utils.wait import TimedOutError
 
 
+# I know this isn't good for reuse, I am just trying to get something to work.
 def _trying_ips(vm, attempts=60, interval=10):
     for attempt in range(attempts):
         for ip in getattr(vm, 'all_ips', []):
-            yield ip
-            return
+            try:
+                if ip.split('.')[0] == '10':
```

Comment:
This is attempt to resolve problem of too-long timeout when IP is not responding. This approach assumes we are dealing with system in RedHat network thus not quite as robust as if this problem was dealt with by passing some sort of `connection_timeout` argument that would be used to set the Paramiko's `timeout` of the `connect` method: http://docs.paramiko.org/en/2.4/api/client.html#paramiko.client.SSHClient.connect

Reply:
This is an attempt to solve the issue of the 192 IPs NEVER allowing an SSH connection wrt this test. We must wait for the attempts to time out. We must sometimes wait a few minutes for the SHH client to come up. If we get the 192 IP first, we'll need to wait for the entire timeout.  I agree this could be handled by increasing the timeout in paramiko and we wouldn't need an additional sleep loop, but we'd still be waiting for 192 to time out.

Reply:
You are right the 192.x address makes delay. But I think you are missing that all ips are meant to be tried in a short while (X), then wait and then again:
```
all_ips returns no IPs .. nothing to do
wait X seconds
all_ips returns no IPs .. nothing to do
wait X seconds
all_ips returns [192.x.y.z] 
   .. attempt to connect 192.x.y.z... timeout after T seconds
wait X secodes
all_ips returns [192.x.y.z, IP_somewhere_in_the_universe]... 
   .. attempt to connect 192.x.y.z, timeout after T seconds
   .. atempt to connect  IP_somewhere_in_the_universe .. perhaps timeout after T seconds as the service is not responding or something strange happening (we will most probably get Connection refused  immediately once the OS is up)
wait X seconds
all_ips returns [192.x.y.z, IP_somewhere_in_the_universe]...
   .. attempt to connect 192.x.y.z, timeout after T seconds
   .. atempt to connect  IP_somewhere_in_the_universe .. connects as service become available
```

Yeah, with more addresses that are time-outing on connections, our delays to try them all become longer. But the timeout (T) for connection attempt doesn't need to be like 60 seconds or whatever is the default but something like several seconds.